scripts/cc-cli/src/commands/setKeys.ts

```diff
@@ -39,7 +39,7 @@ async function setKeysAction(options: OptionValues) {
     keys = options.keys;
   }
 
-  const tx = api.tx.session.setKeys(keys, []);
+  const tx = api.tx.session.setKeys(keys, "");
```

Comment:
Just making sure this is the expected value. `setKeys` does sound like it needs a list.

Reply:
Thats the proof field. It gets either `string`, `Uint8Array` or `Bytes`. After enforced types, empty string seemed the easiest one to provide.